README.md

```diff
@@ -0,0 +1,21 @@
+# It's ALIVE!
+It's ALIVE! is a single-player VR experience in which you play the Doctor's Assistant. Your goal: to bring the doctor's creation to life!
+
+## About
+
+What does this do? You can go more here. Any inspiration? Info dump :D
+
+## Getting Started
+Hey! I want to run this on my PC, how do I do it? 
+
+In order to develop this game, you will need:
```

Comment:
This would go in the pre-requisites, and if you're not sure about higher or lower compatibility, you can just specify the strict current ver you're using.

Reply:
Oh good catch! So the Getting Started section should be more focused on utilizing the package rather than what you need to get it running... I'll make some changes. I think this section would be a good place to outline the Task system, yeah?

Also, totally just specifying my Unity version and moving on from there.

Reply:
The title "Getting Started" can kinda be thought as like.... when you have a binder for school and each section inside is for a different class. It's not a hard rule but that's how I generally see it. 
[This article might help](https://www.freecodecamp.org/news/how-to-write-a-good-readme-file/), "Getting Started" is like part 4, it includes also some generators at the end 🔍